fe/src/components/Table/IssuesTable/IssuesTable.tsx

```diff
@@ -0,0 +1,113 @@
+import Pagination from "@components/Pagination";
+import useFetch from "@hooks/useFetch";
+import { getIssues } from "api";
+import { useIssuesFilter } from "context/IssuesFilterContext";
+import { useCallback, useState } from "react";
+import { styled } from "styled-components";
+import { Table } from "../Table.style";
+import IssuesTableBody from "./IssuesTableBody";
+import IssuesTableHeader from "./IssuesTableHeader";
+
+export default function IssuesTable() {
+  const [pageNumber, setPageNumber] = useState(1);
+  const [selectedIssueIds, setSelectedIssueIds] = useState<Set<number>>(
+    new Set()
+  );
+
+  const { issuesFilter } = useIssuesFilter();
+  const { data: issuesList, reFetch: refetchIssuesList } = useFetch(
+    useCallback(
+      () => getIssues(issuesFilter.text, pageNumber),
+      [issuesFilter.text, pageNumber]
+    )
+  );
+
+  const isAllIssuesSelected =
+    selectedIssueIds.size === issuesList?.data.length &&
+    selectedIssueIds.size > 0;
+
+  const toggleSelectAll = () => {
+    if (isAllIssuesSelected) {
+      deselectAllIssues();
+    } else {
+      selectAllIssues();
+    }
```

Comment:
else 문을 없애서 depth를 한 단계 줄이는 것은 어떨까요!
```suggestion
    if (isAllIssuesSelected) {
      deselectAllIssues();
      return;
    } 
    selectAllIssues();
```